src/rust/ide/src/controller/project.rs

```diff
@@ -6,6 +6,7 @@
 use crate::prelude::*;
 
 use crate::controller::FilePath;
+use crate::controller::visualisation;
```

Comment:
We usually do not do such imports, because we want to see in usages that the handle is a _controller_ handle (and not e.g. module). If you apply my suggestion about alias in src/controller.rs, then you can just write here `controller::Visualisation`